formidable/views.py

```diff
@@ -21,6 +24,86 @@
 from formidable.serializers.presets import PresetsSerializer
 
 
+logger = logging.getLogger(__name__)
+
+
+def extract_function(func_name):
+    """
+    Return a function out of a namespace
+    """
+    func = None
+    if func_name:
+        module, meth_name = func_name.rsplit('.', 1)
+        try:
+            if six.PY3:
+                imported_module = importlib.import_module(module)
+            else:
+                imported_module = importlib.import_module(
+                    module, [meth_name])
+            func = getattr(imported_module, meth_name)
+        except ImportError:
+            logger.error(
+                "An error has occurred impossible to import %s", func_name
+            )
+    return func
+
+
+class CallbackMixin(object):
+    success_callback_settings = ''
+    failure_callback_settings = ''
+    callback_error_message = "An error has occurred with function: `%s`"
+
+    def _call_callback(self, callback):
+        """
+        Tool to simply call the callback function and handle edge-cases.
+        """
+        func = extract_function(callback)
+        # Call function only if existing
+        if not func:
+            return
+        try:
+            func(self.request)
+        except Exception:
+            logger.error(
+                self.callback_error_message, callback
+            )
+
+    def success_callback(self):
+        """
+        Call the failure callback function
+        """
+        callback = getattr(settings, self.success_callback_settings, None)
+        self._call_callback(callback)
+
+    def failure_callback(self):
+        """
+        Call the failure callback function
+        """
+        # Extract the callback function
+        callback = getattr(settings, self.failure_callback_settings, None)
+        self._call_callback(callback)
+
+    def perform_create(self, serializer):
+        response = super(CallbackMixin, self).perform_create(serializer)
+        self.success_callback()
+        return response
+
+    def perform_update(self, serializer):
```

Comment:
I would like to restore the method in the appropriate class. No need `perform_update` on CreateView.